datadog/data_source_datadog_metric_tags.go

```diff
@@ -0,0 +1,59 @@
+package datadog
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/terraform-providers/terraform-provider-datadog/datadog/internal/utils"
+)
+
+func dataSourceDatadogMetricTags() *schema.Resource {
```

Comment:
Some extra food for thought for the metrics team (@nguyening): this data source name is scoped to tags only; would it be preferable to have a more general datasource for metrics metadata, which allows future extension for fields other than tags?

Reply:
@fpighi I defer to you all as the maintainers, but as a user, I'd expect a separate `datadog_metric_metadata` data source, given metric metadata is a different API. So: 

* `datadog_metric_tags` corresponds to [list tags by metric name](https://docs.datadoghq.com/api/latest/metrics/#list-tags-by-metric-name)
* a hypothetical `datadog_metric_metadata` would correspond to [get metric metadata](https://docs.datadoghq.com/api/latest/metrics/#get-metric-metadata)

Reply:
Thanks for the input @mdb (and for moving them to `/fwprovider`). I'm ok accepting this structure.